va/validation-authority.go

```diff
@@ -59,6 +62,43 @@ type verificationRequestEvent struct {
 	Error        string         `json:",omitempty"`
 }
 
+// TODO Update jws.go to accept jose.JsonWebKey in newVerifier
```

Comment:
I'm just going to delete it, because it doesn't even make sense to me now that I look at it.